apps/block_scout_web/lib/block_scout_web/routers/api_router.ex

```diff
@@ -168,6 +168,7 @@ defmodule BlockScoutWeb.Routers.ApiRouter do
       get("/:transaction_hash_param/raw-trace", V2.TransactionController, :raw_trace)
       get("/:transaction_hash_param/state-changes", V2.TransactionController, :state_changes)
       get("/:transaction_hash_param/summary", V2.TransactionController, :summary)
+      get("/:transaction_hash_param/external_transactions", V2.TransactionController, :external_transactions)
```

Comment:
Let's enable this API endpoint only for neon chain type

```suggestion
      if @chain_type == :neon do
        get("/:transaction_hash_param/external_transactions", V2.TransactionController, :external_transactions)
      end
```